docker/run_setup.sh

```diff
@@ -110,22 +110,34 @@ if [ ! -e $dir_prefix/NETWORK_RUNNING ]; then
     # Verify that the transaction status was successful for the deployments
     status=$(cast publish --rpc-url $subnet_a_url $teleporter_deploy_tx |  getJsonVal "['status']")
     if [[ $status != "0x1" ]]; then
-        echo "Error deploying Teleporter Messenger transaction on subnet A."
+        echo "Error deploying Teleporter Messenger on subnet A."
         exit 1
     fi
-    echo "Deployed TeleporterMessenger to Subnet A"
+    echo "Deployed TeleporterMessenger to Subnet A."
     status=$(cast publish --rpc-url $subnet_b_url $teleporter_deploy_tx |  getJsonVal "['status']")
     if [[ $status != "0x1" ]]; then
-        echo "Error deploying Teleporter Messenger transaction on subnet B."
+        echo "Error deploying Teleporter Messenger on subnet B."
         exit 1
     fi
-    echo "Deployed TeleporterMessenger to Subnet B"
+    echo "Deployed TeleporterMessenger to Subnet B."
     status=$(cast publish --rpc-url $subnet_c_url $teleporter_deploy_tx |  getJsonVal "['status']")
     if [[ $status != "0x1" ]]; then
-        echo "Error deploying Teleporter Messenger transaction on subnet C."
+        echo "Error deploying Teleporter Messenger on subnet C."
         exit 1
     fi
-    echo "Deployed TeleporterMessenger to Subnet C"
+    echo "Deployed TeleporterMessenger to Subnet C."
+    status=$(cast publish --rpc-url $c_chain_url $teleporter_deploy_tx |  getJsonVal "['status']")
+    if [[ $status != "0x1" ]]; then
+        echo "Error deploying Teleporter Messenger on C-chain."
+        exit 1
+    fi
+    echo "Deployed TeleporterMessenger to C-chain."
+
+    # Initialize the blockchain ID of the Teleporter contract instances.
+    # TODO: Initialize the C-chain Teleporter contract instance once the warp precompile is added to the C-chain.
+    cast send --private-key $user_private_key $teleporter_contract_address "initializeBlockchainID()" --rpc-url $subnet_a_url
+    cast send --private-key $user_private_key $teleporter_contract_address "initializeBlockchainID()" --rpc-url $subnet_b_url
+    cast send --private-key $user_private_key $teleporter_contract_address "initializeBlockchainID()" --rpc-url $subnet_c_url
```

Comment:
With the `receiveCrossChainMessage` lazy initialization, do we want to remove these?